lib/DXIL/DxilOperations.cpp

```diff
@@ -651,9 +651,14 @@ bool OP::IsDxilOpFuncCallInst(const llvm::Instruction *I, OpCode opcode) {
   return llvm::cast<llvm::ConstantInt>(I->getOperand(0))->getZExtValue() == (unsigned)opcode;
 }
 
+OP::OpCode OP::getOpCode(const llvm::Instruction *I) {
+  return (OP::OpCode)llvm::cast<llvm::ConstantInt>(I->getOperand(0))
+      ->getZExtValue();
+}
+
```

Comment:
It's a nit, but if you end up making another change to this PR anyway, the `IsDxilOpFuncCallInst` function above would look a lot cleaner if it took advantage of this convenience function too.